examples/spark/src/test/scala/org/apache/carbondata/examplesCI/RunExamples.scala

```diff
@@ -120,11 +120,4 @@ class RunExamples extends QueryTest with BeforeAndAfterAll {
   test("DirectSQLExample") {
     DirectSQLExample.exampleBody(spark)
   }
-
-  test("HiveExample") {
```

Comment:
instead of removing, please ignore this test and add a comment why you are ignoring.